src/mpt/utility/getNbOfHeaderLines.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * pass the line and tries to convert the value to integer or
+ * errors out (when NaN).
+ */
+export function getNbOfHeaderLines(line: string): number {
+  const nOfHeaderLines = line.split(' : ');
+  let val = parseInt(nOfHeaderLines[1].trim(), 10);
+  if (isNaN(val)) {
+    throw new Error(
+      'Can not find a numberic value for the number of lines. Aborting.',
```

Comment:
Little spelling mistake. 'numeric'.

I have this extension that I find quite usefull:

![image](https://user-images.githubusercontent.com/1484241/194723925-1d0a1084-4424-4224-8a64-975e593e2a21.png)

Reply:
Fixed. I'll install the extension thanks